src/Directory.Build.props

```diff
@@ -1,9 +1,12 @@
 <Project>
+  <!-- MSBuild only includes the first Directory.Build.props, so we need to manually include the root one -->
+  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
```

Comment:
Maybe `/` like above?

Reply:
msbuild does a normalization, the relevant change was the casing. I think I will go back to backslash for everything

Reply:
> Maybe we should add that as comment then?

I had a comment in the actual commit, but yes, will add a source comment